packages/ui/src/common/components/MarkdownPreview/MarkdownPreview.tsx

```diff
@@ -62,11 +62,12 @@ export const MarkdownPreview = ({ markdown, append, ...styleProps }: MarkdownPre
     }
   }, [markdown, append])
 
+  const stripSlashes = (text: string) => text.replace(/\\(.)/gm, '$1')
   return (
     <div className="markdown-preview">
       <MarkdownPreviewStyles {...styleProps} />
-      <ReactMarkdown rehypePlugins={rehypePlugins} components={components} rawSourcePos>
-        {markdown}
+      <ReactMarkdown rehypePlugins={rehypePlugins} components={components} rawSourcePos skipHtml unwrapDisallowed>
+        {stripSlashes(markdown)}
       </ReactMarkdown>
```

Comment:
nice work!
wonder if tables in https://pioneer-2-git-fork-oleksanderkorn-bugfix-2712-joystream.vercel.app/#/forum/thread/57 work with [gfm](https://blog.logrocket.com/how-to-safely-render-markdown-using-react-markdown/), also some of the `## headings`
```
import gfm from 'remark-gfm'
<ReactMarkdown remarkPlugins={[gfm]}
```

Reply:
Seems to be not working locally, also tried here with enabled gfm, but no luck: https://remarkjs.github.io/react-markdown/
<img width="1501" alt="image" src="https://user-images.githubusercontent.com/9252017/171138942-0ca7950f-9c55-4695-94fb-4c9b18308bb1.png">